erudio-client.json

```diff
@@ -0,0 +1,4 @@
+{
+  "erudio_base_url":"https://api.dev.edtech.fagbokforlaget.no/",
+  "erudio__access_token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2MGMzMTY0ZTYyNGRjNWUyNGJjYjBhZWUiLCJyb2xlcyI6WyJzdHVkZW50Il0sImV4dGVybmFsX2lkIjoibjZhWWV1dzNZN3N3VkhSTVpvMHFIeE5Wb19FdE1YNVZ6VUJxVmJaZllwYyIsImF1ZCI6ImxvY2FsaG9zdDo4MDAwIiwiaXNzIjoiYXV0aC1wcm94eS5kZXYuZWR0ZWNoLmZhZ2Jva2ZvcmxhZ2V0Lm5vIiwicHJpbWFyeV9yb2xlIjoic3R1ZGVudCIsInByb3ZpZGVyIjoiYXp1cmVhZCIsInRlbmFudCI6ImZvcmxhZ3NodXNldCIsImlhdCI6MTY2MTQwNzgyMiwiZXhwIjoxNjYyNjE3NDIyfQ.qYMzjTYkqwH5fj4S_sC9zpS7Fnpqmif8g3dCbT7EIQc"
```

Comment:
@JERILJOSE96 Oups! This token should be revoked immediately, as it has been exposed in git history.

Reply:
yeah. accidentally commited. removed